src/lib/crosslink.js

```diff
@@ -0,0 +1,9 @@
+import { compressToBase64 } from 'lz-string';
+
+const version = 'lz';
```

Comment:
I'm a little doubtful about this name.
Maybe instead of _lz_, use something more reflecting the meaning of the link, for example, _external_?
_lz_ is the implementation details. The implementation may change over time.

Reply:
Yep? it's main idea to be ready for change encoding type or json schema. If we use single entrypoint like `/external/` it's to difficult detect what format is inside base64 (it's important for backward compatibility). If you don't like it, we can use `/share/v1/:hash:` or smthg with id scheme+format.

Reply:
I agree, let's keep your version.